src/lib/modal.svelte

```diff
@@ -0,0 +1,379 @@
+      picture: '/modal/cat4.png',
+      resourceTexts: ['TNR'],
+      resourceLinks: ['https://www.youtube.com/watch?v=IarsSTLUdTw']
+    },
+    {
+      title: 'step 5: HOLD',
+      subtitle: 'Hold the cat overnight until surgery the next day',
+      info: 'Congrats, you got the cat! Cover the trap and place it in a warm, quiet place overnight (like a bathroom or a heated garage). Check the cat once an hour, every hour, until bedtime.',
+      picture: '/modal/cat5.png',
+      resourceTexts: ['Pre-op Tips'],
+      resourceLinks: ['https://fixnation.org/about-tnr/pre-post-surgery-instructions/']
+    },
+    {
+      title: 'step 6: FIX',
+      subtitle: 'Take the cat to get fixed (for free!).',
+      info: 'Feed kittens a teapoon of wet food at 6 am on surgery day; no food or water for older cats. Cover the car seats with plastic and securely place the traps on top. Arrive at FixNation between 7-8.30 am. Leave kitty in car and wait in line to check-in. FixNation will spay/neuter the cat and provide vaccines, flea treatment, fluids, pain medication and an ear tip. Pick-up from 3-4.30 pm.',
+      picture: '/modal/cat6.png',
+      resourceTexts: ['Check-in Process'],
+      resourceLinks: ['https://www.facebook.com/LuxePaws/videos/3379488712127276/']
+    },
+    {
+      title: 'step 7 WATCH:',
+      subtitle: 'Keep the cat one more night for recovery.',
+      info: 'Take kitty home in the covered trap and keep in a warm, quiet place overnight. Feed a ¼ can of wet food about six hours post-op (check paperwork for exact time). Check kitty once an hour, every hour, until bedtime. ',
+      picture: '/modal/cat7.png',
+      resourceTexts: ['Post-ops tips'],
+      resourceLinks: ['https://www.alleycat.org/community-cat-care/post-surgery-care/']
+    },
+    {
+      title: 'step 8: RETURN',
+      subtitle: 'Return the cat to where it was trapped.',
+      info: 'This is the big moment…release the cat where you trapped it the morning after surgery. Freedom!!!! Clean the trap and trap cover and return to FixNation during opening hours.',
+      picture: '/modal/cat8.png',
+      resourceTexts: ['Community Cat Video'],
+      resourceLinks: ['https://www.youtube.com/watch?v=3-pK4sjVHcQ']
+    }
+  ];
+</script>
+
+{#if open}
+  <div class="modal">
+    <!-- svelte-ignore a11y-click-events-have-key-events -->
+    <div
+      class="cover"
+      on:click={() => {
+        open = false;
+      }}
+    />
+    {#each entries as entry, index}
+      {#if cur == index}
+        <div class="content" in:fly={{ x: dir ? 50 : -50 }} out:fly={{ x: dir ? -50 : 50 }}>
+          <!-- svelte-ignore a11y-click-events-have-key-events -->
+          <div
+            class="x-out"
+            on:click={() => {
+              open = !open;
```

Comment:
I think we have to flip open when the user clicks out of the modal as well instead of using the X